### PR TITLE
Add Pet Namer Tutorial

### PR DESCRIPTION
**Pull request by DoubleA-MG**

### CODE REVIEW

**Review by CommanderFoo — APPROVED** [review]
Applied the changes myself, only small things :)

**Comment by DoubleA-MG** [comment]
Made some small changes I noticed myself. Also you added "Entry" to a regular UI Text Box object.

**Comment by CommanderFoo** [comment]
My bad. Stupid search and replace :P

File: src/tutorials/pet_namer.md [hunk 1]
```diff
@@ -316,4 +316,4 @@ The **UI Text Entry Box** is a really exciting tool. It allows creators to not r
 
 ## Learn More
 
-[UI Text Entry Box Reference](../references/ui_text_entry_box.md) | [UI Text Entry Box API](../api/uieditabletext.md) |
+[Text Entry Validation Tutorial](../tutorials/text_entry_validation/) | [UI Text Entry Box Reference](../references/ui_text_entry_box.md) | [UI Text Entry Box API](../api/uieditabletext.md) |
```
Comment (CommanderFoo) [inline src/tutorials/pet_namer.md]: Add UI for consistency

Comment (CommanderFoo) [reply src/tutorials/pet_namer.md]:
```suggestion
[UI Text Entry Validation Tutorial](../tutorials/text_entry_validation.md) | [UI Text Entry Box Reference](../references/ui_text_entry_box.md) | [UI Text Entry Box API](../api/uieditabletext.md) |
```

**Review by CommanderFoo — APPROVED** [review]

File: src/tutorials/pet_namer.md (hunk 1 above)
Comment (DoubleA-MG) [reply src/tutorials/pet_namer.md]: Good catch

**Pull request merged**